src/Application.php

```diff
@@ -325,6 +326,7 @@ public function getCommands()
             new Cmd\PullRequest\PullRequestCheckoutCommand(),
             new Cmd\Util\DocumentationCommand(),
             new Cmd\Util\FabbotIoCommand(),
+            new Cmd\Util\StyleCiPatchCommand(),
```

Comment:
Like in our namespaces: https://github.com/StyleCI/StyleCI/blob/master/app/Events/Analysis/AnalysisHasCompletedEvent.php#L12.